### ISSUE TITLE
Call to undefined function PhpBrew\json_decode() in phar:///usr/bin/phpbrew/PhpBrew/ReleaseList.php on line 2

### ISSUE THREAD

**Issue by rxt360** [phpbrew/phpbrew#491]
When I execute the following command, I get an error about json:

    phpbrew install 5.4.34 +default +gd +json +apxs2=/usr/bin/apxs2
    PHP Fatal error:  Call to undefined function PhpBrew\json_decode() in phar:///usr/bin/phpbrew/PhpBrew/ReleaseList.php on line 2
    Fatal error: Call to undefined function PhpBrew\json_decode() in phar:///usr/bin/phpbrew/PhpBrew/ReleaseList.php on line 2

System: Ubuntu 14.04, PHP 5.5 installed with json support, but apparantly this is for the system PHP, not for phpbrew. PHPbrew works and I now have phpbrew-5.4.34 working for CLI and Apache. 

    php -v
    PHP 5.4.34 (cli) (built: Apr 21 2015 12:52:01)

Any ideas how to fix this?

**Comment by c9s** [comment]
looks like your system php does not support json? json extension is installed ?

**Comment by rxt360** [comment]
PHP-json is installed AFAIK. It's installed by default I believe. 

    sudo apt-get install php5-json
    php5-json is already the newest version.

**Comment by c9s** [comment]
Can you paste the output of the commands below?

```sh
which php
php -r 'var_dump(json_decode("{}"));';
```

**Comment by rxt360** [comment]
Yes - thanks for the quick reply:

    which php
    /root/.phpbrew/php/php-5.4.34/bin/php

And:

    php -r 'var_dump(json_decode("{}"));';
    PHP Fatal error:  Call to undefined function json_decode() in Command line code on line 1
    Fatal error: Call to undefined function json_decode() in Command line code on line 1

**Comment by c9s** [comment]
I think you need to set `phpbrew off` and use the system php to build another php :)

**Comment by rxt360** [comment]
I turned phpbrew off, then turned if off for Apache, chaning libphp5.4.34.so to libphp5.so.

    vim /etc/apache2/mods-enabled/php5.load
    LoadModule php5_module        /usr/lib/apache2/modules/libphp5.4.34.so

Then I removed the complete ~/.phpbrew folder, and started over.

    phpbrew install 5.4.34 +default +gd +json +pgsql +apxs2=/usr/bin/apxs2

It all works now! 

NB: It took quite some googling to find out how to disable phpbrew in apache after using +apx2. Maybe I looked in the wrong places, but I think it would be helpful to have more documentation about this. 

Thanks for the support!
